pages/common/mongodump.md

```diff
@@ -0,0 +1,27 @@
+# mongodump
+
+> Utility to export the contents of a MongoDB instance.
+
+- Take a dump of all databases. (This will place the files inside a folder called "dump"):
```

Comment:
This is more of a stylistic preference, but I would prefer the parenthesised expression to be removable, which would leave us with a `.:` -- instead, how about `Take a dump of all databases (this will place the files inside a folder called "dump"):`?